ArjunaJTA/jdbc/pom.xml

```diff
@@ -196,89 +196,5 @@
               </plugins>
           </build>
       </profile>
-    <profile>
```

Comment:
@mmusgrov I see this message in the commit history: <!-- j9 TODO Table "TEST_TABLE" already exists and ClassNotFound oracle.jdbc.xa.client --> do you think its the version of Oracle we test with?